lib/fennec/turn.ex

```diff
@@ -9,6 +9,7 @@ defmodule Fennec.TURN do
     nonce: nil, realm: nil
 
   @permission_lifetime 5 * 60 # MUST be 5mins
+  @channel_lifetime 10 * 60 # 10 minutes
```

Comment:
BTW. Why don't we use `:timer.minutes(10)` here (or some Elixir's / our wrapper around it)? The comment could be shorter :)